openage/convert/gamedata/unit.py

```diff
@@ -719,9 +813,9 @@ class UnitObject(Exportable):
             },
         )),
         (READ, "editor_selection_color", "uint8_t"),  # 0: default, -16: fish trap, farm, 52: deadfarm, OLD-*, 116: flare, whale, dolphin -123: fish
-        (READ_EXPORT, "selection_shape_x", "float"),
-        (READ_EXPORT, "selection_shape_y", "float"),
-        (READ_EXPORT, "selection_shape_z", "float"),
+        (READ_EXPORT, "outline_size_x", "float"),
+        (READ_EXPORT, "outline_size_y", "float"),
+        (READ_EXPORT, "outline_size_z", "float"),
```

Comment:
i think this is the circle or square when a unit is selected, not the outline (which is the pixel-correct background-unit border).